VLApi/Program.cs

```diff
@@ -0,0 +1,33 @@
+using Microsoft.EntityFrameworkCore;
+using VL.Shared.Data;
+using VL.Shared.Model;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+builder.Services.AddDbContext<ApplicationDbContext>(options => options.UseSqlServer(
+    builder.Configuration.GetConnectionString("DefaultConnection")
+));
+
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();
+
+// Configure the HTTP request pipeline.
+if (app.Environment.IsDevelopment())
+{
+    app.UseSwagger();
+    app.UseSwaggerUI();
+}
+
```

Comment:
You still need to add migrations and seeding for development

Imagine the other project "VirtualLibrary" does not exist, they just happen to share the same solution

Reply:
No database exists, clean slate when a user first runs this project